setup.py

```diff
@@ -514,7 +514,12 @@ def get_common_options(build_ext):
         LIBRARIES += ['ddl', 'ddl_pack']
 
     if mxnet_include_dirs:
-        MACROS += [('MSHADOW_USE_CBLAS', '1')]
+        #MACROS += [('MSHADOW_USE_CBLAS', '1')]
+        if have_cuda:
+            MACROS += [('MSHADOW_USE_CUDA', '1')]
+        else:
+            MACROS += [('MSHADOW_USE_CUDA', '0')] 
+        MACROS += [('MSHADOW_USE_MKL', '0')]
```

Comment:
why set MSHADOW_USE_MKL to 0 here?

Reply:
Because my MacBook does not have MKL library and failed to build. I will add a function to check the existence of MKL package here.